libdnf/conf/ConfigMain.cpp

```diff
@@ -185,6 +185,7 @@ class ConfigMain::Impl {
     OptionString logdir{"/var/log"};
     OptionNumber<std::int32_t> log_size{1024 * 1024, strToBytes};
     OptionNumber<std::int32_t> log_rotate{4, 0};
+    OptionBool log_compression{false};
```

Comment:
How about naming it `log_compress` to keep consistency with `log_rotate` (using a verb instead of a noun)?